### ISSUE TITLE
Update repo to use newer biopython version

### ISSUE THREAD

**Issue by chg60** [SEA-PHAGES/pdm_utils#20]
Biopython has made some significant changes in recent versions, the most important of which for our purposes is the handling of alphabets.

**Comment by edgingtonn1** [comment]
This is crashing 'import' correct?

```
python3 -m pdm_utils import Actino_Draft ./genomes/ import_table.csv -o ./

Traceback (most recent call last): 
File "/opt/anaconda3/envs/pdm_utils_env/lib/python3.8/site-packages/Bio/Alphabet/__init__.py", line 20, in <module>
    raise ImportError(
ImportError: Bio.Alphabet has been removed from Biopython. In many cases, the alphabet can simply be ignored and removed from scripts. In a few cases, you may need to specify the ``molecule_type`` as an annotation on a SeqRecord for your script to work correctly. Please see https://biopython.org/wiki/Alphabet for more information.
```

**Comment by chg60** [comment]
Hi Nick,

Yes, that is very likely the reason for crashing the import pipeline. I'm curious if you can share some details about your environment - how did you install pdm_utils, and what versions of pdm_utils and Biopython do you have installed right now?

In theory if you installed using pip anytime recently, a compatible version of Biopython should have been installed, so I'm a little nervous to see that you got this crash...

Thanks!

-Christian

**Comment by edgingtonn1** [comment]
Hi Christian,

Here are my environment packages:
```
❯ conda --version
conda 4.13.0
```
[pdm_utils_env.txt](https://github.com/SEA-PHAGES/pdm_utils/files/9437091/pdm_utils_env.txt)

Best,
-Nick

**Comment by chg60** [comment]
Yep, looks like your biopython version is too high. The only way I can think that this might have happened is if you ran something like 'conda update --all', as the pip installer would have freaked out about installing an incompatible package version.

Can you try:

pip uninstall pdm_utils biopython
pip install pdm_utils

And see whether that fixes it?  The first should remove the bad version of biopython in a manner that pip won't complain about (since we are simultaneously removing the package that requires it), and then when re-installing pdm_utils it should pull biopython==1.77 and work properly from then on.

**Comment by edgingtonn1** [comment]
Yes, that's on me! I did happen to 'conda update --all'. Going to Biopython 1.77 removed the error. Thanks!

**Comment by chg60** [comment]
Awesome, thanks Nick!
